app/javascript/tests/public_health/bulk_actions/CloseRecords.test.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import CloseRecords from '../../../components/public_health/bulk_actions/CloseRecords';
+import { mockPatient1, mockPatient2, mockPatient6 } from '../../mocks/mockPatients';
+import { mockMonitoringReasons } from '../../mocks/mockMonitoringReasons';
+
+const onCloseMock = jest.fn();
+const mockToken = 'testMockTokenString12345';
+
+function getWrapper(patientsArray) {
+  let wrapper = shallow(<CloseRecords authenticity_token={mockToken} patients={patientsArray} close={onCloseMock} monitoring_reasons={mockMonitoringReasons} />);
+  return wrapper;
+}
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+describe('CloseRecords', () => {
+  it('Sets intial state after mount correctly', () => {
+    const wrapper = getWrapper([mockPatient1, mockPatient2, mockPatient6]);
+    expect(wrapper.state('apply_to_household')).toBeFalsy();
+    expect(wrapper.state('loading')).toBeFalsy();
+    expect(wrapper.state('monitoring')).toBeFalsy();
+    expect(wrapper.state('monitoring_reason')).toEqual('');
+    expect(wrapper.state('reasoning')).toEqual('');
+  });
+
+  it('Properly renders all main components', () => {
+    const wrapper = getWrapper([mockPatient1, mockPatient2, mockPatient6]);
+    expect(wrapper.find('p').find('span').at(0).text()).toEqual('You are about to change the Monitoring Status of the selected records from "Actively Monitoring" to "Not Monitoring".');
+    expect(wrapper.find('p').find('span').at(1).text()).toEqual(' These records will be moved to the closed line list and the reason for closure will be blank.');
+    expect(wrapper.find('FormGroup').length).toEqual(3);
```

Comment:
Overall comment but can we change all the Modal/Form finds from `FormGroup` to Form.Group to be consistent with the rest of the test files?